docs/user_guide/event_processors/event_value_tag.md

```diff
@@ -0,0 +1,179 @@
+                "interface_name": "Ethernet1",
+                "description": "Uplink"
+            },
+            "values": {
+                "/interfaces/interface/status/counters/in-octets": 100
+            }
+        },
+        {
+            "name": "sub1",
+            "timestamp": 200,
+            "tags": {
+                "source": "leaf1:6030",
+                "subscription-name": "sub1",
+                "interface_name": "Ethernet1",
+                "description": "Uplink"
+            },
+            "values": {
+                "/interfaces/interface/status/counters/out-octets": 100
+            }
+        },
+        {
+            "name": "sub1",
+            "timestamp": 200,
+            "tags": {
+                "source": "leaf1:6030",
+                "subscription-name": "sub1",
+                "interface_name": "Ethernet1"
+            },
+            "values": {
+                "/interfaces/interface/status/description": "Uplink"
+            }
+        }
+    ]
+    ```
+
+```yaml
+  bgp-description:
+    event-value-tag:
+      value-name: "neighbor_description"
+      consume: true
+```
+=== "Event format before"
+    ```json
+    [
+        {
+            "name": "sub2",
+            "timestamp": 1615284691523204299,
+            "tags": {
+                "neighbor_peer-address": "2002::1:1:1:1",
+                "network-instance_name": "default",
+                "source": "leaf1:57400",
+                "subscription-name": "sub2"
+            },
+            "values": {
+                "bgp_neighbor_sent_messages_queue_depth": 0,
+                "bgp_neighbor_sent_messages_total_messages": "423",
+                "bgp_neighbor_sent_messages_total_non_updates": "415",
+                "bgp_neighbor_sent_messages_total_updates": "8"
+            }
+        },
+        {
+            "name": "sub2",
+            "timestamp": 1615284691523204299,
+            "tags": {
+                "neighbor_peer-address": "2002::1:1:1:1",
+                "network-instance_name": "default",
+                "source": "leaf1:57400",
+                "subscription-name": "sub2"
+            },
+            "values": {
+                "neighbor_description": "PeerRouter"
+            }
+        }
+    ]
+    ```
+=== "Event format after"
+    ```json
+    [
+        {
+            "name": "sub2",
+            "timestamp": 1615284691523204299,
+            "tags": {
+                "neighbor_peer-address": "2002::1:1:1:1",
+                "network-instance_name": "default",
+                "source": "leaf1:57400",
+                "subscription-name": "sub2"
+            },
+            "values": {
+                "bgp_neighbor_sent_messages_queue_depth": 0,
+                "bgp_neighbor_sent_messages_total_messages": "423",
+                "bgp_neighbor_sent_messages_total_non_updates": "415",
+                "bgp_neighbor_sent_messages_total_updates": "8",
+                "neighbor_description": "PeerRouter"
```

Comment:
shouldn't `neighbor_description` be in the tags map in this example ?

Reply:
You are correct.  Fixed/pushed.